src/app.js

```diff
@@ -462,6 +462,7 @@ var run = function () {
       $('#ghostbar').remove()
       $(document).unbind('mousemove')
       dragging = false
+      delta = (delta < 50) ? 50 : delta
```

Comment:
Yes, would be enough, but probably less user friendly because his action is always reset to the prior state if he leaves the window while dragging. Apart form that he could drag very close to 0 making it hard to make the bar appear again. For hidding it there is a button anyways.